utils/rpms/daos.spec

```diff
@@ -166,12 +166,10 @@ Requires: ndctl
 # needed to set PMem configuration goals in BIOS through control-plane
 %if (0%{?suse_version} >= 1500)
 Requires: ipmctl >= 02.00.00.3733
-# When 1.11.2 is released, we can change this to >= 1.11.2
-Requires: libpmemobj1 = 1.11.0-3.suse1500
+Requires: libpmemobj1 >= 1.12.1~rc1-1.suse1500
```

Comment:
Didn't we run into an issue in prior iterations of this update where we discovered that we were only setting a version on this requires for this one library but that this RPM also Requires others from pmdk that we should be setting the version on?

Reply:
so you think we should be adding libpmem and libpmemblk ?

Reply:
Is there any way to tell it to use libpmemobj1 >= [version] and ensure that libpmem and libpmemblk have same version ?

Reply:
though I assume it's a little different here since we are using >= and I presume we wouldn't get mixed versions again

Reply:
Ah, yes.  I suppose perhaps that issue previously was just with having the one pinned and not the rest.  So indeed, perhaps we can just rely on pinning one of them and the others will be the same version.  Although, quite honestly, I'd just rather remove the constraint altogether and rely on proper repo contents to get us what we want.  But I understand some people like the constraint.  Even if an improper repo configuration will cause the install to break anyway.